### ISSUE TITLE
Is mailgun.org still vulnerable??

### ISSUE THREAD

**Issue by PH-Apolonio** [EdOverflow/can-i-take-over-xyz#393]
## Service name
mailgun.org
## Proof
![Screenshot 2023-11-14 182105](https://github.com/EdOverflow/can-i-take-over-xyz/assets/74172849/17cc0c32-d4a3-4b49-8aaf-26f2497c15f0)

## Documentation
I have this target that the cname is pointing to  mailgun.org and i try to taking over it, but the only verified is the SPF and MX records , the DKIM and cname was not verified, so the question is , is this still vulnerable??

**Comment by muslimfrompk** [comment]
No I have a similar target but while trying to register the give a response like this:
`HTTP/2 400 Bad Request
Access-Control-Allow-Credentials: true
Access-Control-Allow-Origin: *
Cache-Control: no-store
Content-Type: application/json; charset=utf-8
Date: Sat, 18 Nov 2023 16:39:03 GMT
Strict-Transport-Security: max-age=63072000; includeSubDomains
X-Mailgun-Key-Id: 1c7e8847-e517f0dd
X-Xss-Protection: 1; mode=block
Content-Length: 72

{"message":"Invalid parameter: name invalid ascii character found ' '"}
`

**Comment by muslimfrompk** [comment]
So it does not seem to be vulnerable in our case

**Comment by PH-Apolonio** [comment]
thanks for that

**Comment by vegeta2op** [comment]
Yes , It's MX takeover but the mail lands in spam + you can only send it to yourself and the users you have verified

**Comment by felipecaon** [comment]
They requiring a custom TXT now

**Comment by jbmihoub** [comment]
so no mailgun takeover anymore ?

**Comment by felipecaon** [comment]
No
